lake-primitives/src/types/events.rs

```diff
@@ -26,6 +30,19 @@ impl Event {
     pub fn related_receipt_id(&self) -> &crate::CryptoHash {
         &self.related_receipt_id
     }
+
+    pub fn related_receipt_receiver_id(&self) -> &AccountId {
```

Comment:
I cannot leave comments to the code above, but I suggest you update the `event`, `standard`, `version`, and `data` to return references to the data instead of cloning.

`related_receipt_id` can return an owned CryptoHash since it is a copy-type anyway.